src/main/java/com/xjeffrose/chicago/AggregatedChicagoMessage.java

```diff
@@ -70,7 +70,7 @@ public void appendVal(byte[] valToAppend) {
     byte[] newVal = new byte[val.length + valToAppend.length + 1];
 
     System.arraycopy(val, 0, newVal, 0, val.length);
-    System.arraycopy(",".getBytes(), 0, newVal, val.length, 1);
+    System.arraycopy("\0".getBytes(), 0, newVal, val.length, 1);
```

Comment:
Don't forget to update the test: https://github.com/xjdr/chicago/blob/fe87504914f7ae40b370b6832d1704726fa8744e/src/test/java/com/xjeffrose/chicago/ChicagoMessageAggregatorTest.java#L170